hugo/content/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: About
+
+---
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The source code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip/).
```

Comment:
GitHub itself links to org without slash, so we should use that. Strangely enough slash does not redirect to without.

```suggestion
For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip).
```